RamaLama_rag_gyrc9nqe/RamaLama_rag_gshthiz3/.lock

```diff
@@ -0,0 +1 @@
+tmp lock file
```

Comment:
Need to get rid of the junk files

Reply:
If this branch is rebased after you remove the leftover files should get past the build issues